pkg/gardenlet/operation/botanist/vpa.go

```diff
@@ -83,7 +83,6 @@ func (b *Botanist) DefaultVerticalPodAutoscaler() (vpa.Interface, error) {
 		b.SecretsManager,
 		vpa.Values{
 			ClusterType:              component.ClusterTypeShoot,
-			Enabled:                  true,
```

Comment:
Can we add an unit test about it? When VPA is disabled in the Seed spec, the returned vpa component's Deploy does Destroy.